static/styles/subscriptions.css

```diff
@@ -866,90 +858,6 @@ h4.stream_setting_subsection_title {
         }
     }
 
-    .stream-header {
-        white-space: nowrap;
-
-        .stream-name {
-            display: inline-block;
-            position: relative;
-            font-size: 1.5em;
-            font-weight: 600;
-            margin-left: -3px;
-            margin-top: -5px;
-            white-space: nowrap;
-            max-width: 260px;
-
-            .sub-stream-name {
-                display: block;
-                max-width: 20ch;
-                overflow: hidden;
-                text-overflow: ellipsis;
-                float: left;
-            }
-        }
-
-        .button-group {
-            float: right;
-
-            .deactivate {
-                margin-right: 3px;
-                text-decoration: none;
-            }
-        }
-
-        .subscribe-button.unsubscribed {
-            color: hsl(156, 39%, 54%);
-            border-color: hsl(156, 39%, 77%);
-
-            &:active {
-                background-color: hsla(156, 39%, 54%, 0.2);
-            }
-        }
-
-        .large-icon {
-            display: inline-block;
-            vertical-align: top;
-            margin-right: 5px;
-            margin-top: -5px;
-            font-size: 1.5em;
-
-            &.hash::after {
-                top: 1px;
-                font-size: 1.09em;
-                font-weight: 800;
-            }
-        }
-
-        .stream_change_property_info {
-            vertical-align: top;
-            margin: -5px 0 0;
-        }
-    }
-
-    .stream-description {
-        margin-bottom: 5px;
-
-        .no-description {
-            font-style: italic;
-            color: hsl(0, 0%, 67%);
-        }
-    }
-
-    .stream_setting_subsection_header {
-        display: flex;
-
-        .stream_permission_change_info {
-            /* TODO: Deduplicate with .save-discard-widget-button. */
-            margin: 12px auto 12px 3px;
-        }
-
-        .button-group {
-            display: inline;
-            margin-left: auto;
-            align-self: center;
-        }
-    }
-
```

Comment:
all these 3 classes `.stream-header`, `.stream-description` ,`.stream_setting_subsection_header`
exists nested inside `subscription_settings` -> `stream_section`;

I've added their CSS nested inside the `stream_section`.